presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeTpcdsQueries.java

```diff
@@ -73,6 +73,7 @@ private static void createTpcdsCallCenter(QueryRunner queryRunner, Session sessi
         if (!queryRunner.tableExists(session, "call_center")) {
             switch (storageFormat) {
                 case "PARQUET":
+                case "ORC":
```

Comment:
DWRF didn't support DATE types and so the CREATE SQL for DWRF casts date columns to Varchar. OrcReader is using DwrfReader under the hood. So I'm curious how it is supporting Date types. Please can you elaborate.

Reply:
DWRF does not support the DATE type, but Velox queries ORC's DATE type using SelectiveIntegerDirectColumnReader. My test shows that the DATE type data can be read correctly. So I don't think it is necessary to convert the DATE type to VARCHAR.

Reply:
I recently added a parameter to createAllTables to not do the DATE -> VARCHAR casting for TPCH tables https://github.com/prestodb/presto/blob/master/presto-native-execution/src/test/java/com/facebook/presto/nativeworker/NativeQueryRunnerUtils.java#L69. You can use it in your tests.